Framework/Kernel/test/MersenneTwisterTest.h

```diff
@@ -139,10 +139,10 @@ class MersenneTwisterTest : public CxxTest::TestSuite {
 
 private:
   void assertSequenceCorrectForSeed_39857239(MersenneTwister &randGen) {
-    double expectedValues[10] = {
-        0.203374452656, 0.597970068222, 0.120683325687,  0.92372657801,
-        0.734524340136, 0.467380537419, 0.0712658402044, 0.204503614921,
-        0.487210249063, 0.885743656661};
+    double expectedValues[10] = {0.597970068269, 0.923726578038, 0.46738053759,
```

Comment:
Do we know why the values change for the same seed given that it is supposed to be the same random number generator type?